openshift_cli_installer/libs/clusters/rosa_cluster.py

```diff
@@ -47,7 +47,10 @@ def __init__(self, ocp_cluster: Dict[str, Any], user_input: UserInput) -> None:
         if not self.user_input.destroy_from_s3_bucket_or_local_directory:
             if self.cluster_info["platform"] == HYPERSHIFT_STR:
                 self.terraform = Terraform()
-                self.cluster["tags"] = "dns:external"
+                user_tags = self.cluster_info.get("tags", "")
+                if user_tags:
```

Comment:
nit:
```suggestion
                if user_tags := self.cluster_info.get("tags")
```
no need to set to an empty string as default

Reply:
it is needed, otherwise we would concatenate "None"